lib/config/definitions.ts

```diff
@@ -308,6 +308,14 @@ const options: RenovateOptions[] = [
     admin: true,
     type: 'string',
   },
+  {
+    name: 'customEnvVariables',
+    description:
+      'Custom environment variables for child processes and sidecar Docker containers. This configuration will be applied after all other environment variables so that it can be used to override defaults.',
```

Comment:
```suggestion
      'Custom environment variables for child processes and sidecar Docker containers.',
```

The rest will be in the markdown doc